server/src/routes/bugs.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const bugsController = require('../controllers/bugs');
+const router = express.Router();
+
+// sample test route to just get all the STATUS values
+router.get('/getAllStatus', bugsController.getAllStatus, (req, res) => res.status(200).json(res.locals.data));
+
+// TODO: /create
+// this route will create a new bug in the database
+router.post('/create', bugsController.create, (req, res) => res.status(200).json(res.locals.data));
+
+// TODO: /details/:bugId
+// this route will get the current bug details for given bug id
+
+// TODO: /update/:bugId
+// this route will update the bug details for given bug id
+
+// TODO: /delete/:bugId
+// this route will delete the bug for given bug id
+
+// TODO: /getAssigned/:userId
```

Comment:
I think this route could also be expressed in User related route.
For example: /api/users/:userID/bugs

Reference (see bullet point number 5):
https://medium.com/hashmapinc/rest-good-practices-for-api-design-881439796dc9